openshift_pipelines/op-release-notes.adoc

```diff
@@ -0,0 +1,41 @@
+//OpenShift Pipelines Release Notes
+include::modules/openshift-pipelines-attributes.adoc[]
+[id="op-release-notes"]
+= {op-title} Release Notes
+:context: op-release-notes
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+{op-title} is a cloud-native CI/CD experience based on the link:https://github.com/tektoncd[Tekton] project which provides:
+
+* Standard Kubernetes-native pipeline definitions (CRDs).
+* Serverless pipelines with no CI server management overhead.
+* Extensibility to build images using any Kubernetes tool, such as S2I, Buildah, JIB, and Kaniko.
+* Portability across any Kubernetes distribution.
+* Powerful CLI for interacting with pipelines.
+* Integrated user experience with the OpenShift Console developer perspective.
+
+////
+For an in-depth walk-through of {op-title}, see xref:../openshift_pipelines/understanding-openshift-pipelines.adoc [Understanding OpenShift Pipelines].
+////
+[id="getting-support"]
+== Getting support
+
+If you experience difficulty with a procedure described in this documentation,
+visit the link:https://access.redhat.com/support/offerings/techpreview/[Customer Portal] to learn more about support for Technology Preview features.
+
+=== Getting in touch with the product team
+For questions and feedback send an email to pipelines-interest@redhat.com mailing list or join the link:https://coreos.slack.com/messages/CG5GV6CJD[#tektoncd-pipelines] channel on CoreOS Slack workspace. You can also join the link:http://tektoncd.slack.com/[Tekton Slack workspace]  for Tekton specific queries.
```

Comment:
CoreOS Slack is internal one. I'm not sure about the mailing list

Reply:
@siamaksade - Could you please confirm. Thanks

Reply:
Yes, CoreOS is internal

Reply:
Updated this section to include only the email